dart/CHANGELOG.md

```diff
@@ -1,5 +1,10 @@
 # Changelog
 
+## 3.15.0-dev.0
```

Comment:
Can do - I don't actually agree that this is the correct thing to do though longer term. The version of the protocol doesn't have a perfect correlation to the semantic versioning of this package, which should be related to the generated apis and helper libraries surrounding the protocol.

For instance, assuming we move forward with the server support proposal, any additional method added to the protocol would be a breaking change in this package (but not the protocol), because the abstract class defining the service would have additional methods which must be implemented (or else you would get compilation errors).

Reply:
Yeah, I get your reservations. The logic here isn't entirely clear. Ideally I'd like clients to be aware of what version of the protocol they're using. Perhaps there's another method besides encoding it as the main part of the version.

For codemirror, I encode it as the version suffix: https://github.com/google/codemirror.dart/blob/master/pubspec.yaml#L6.